src/ResourceManager/KeyVault/Commands.KeyVault.Test/ScenarioTests/KeyVaultManagementTests.cs

```diff
@@ -182,7 +182,7 @@ public void TestGetVaultByNameCapitalized()
                 );
         }
 
-        [Fact]
+        [Fact(Skip = "The current SynchronizationContext may not be used as a TaskScheduler.")]
```

Comment:
@markcowl there are several tests disabled and these tests are mainly useless as we have a different way of testing. However, I try to address them. The link doesn't mean anything to me. When I trace code it gets there but how should I fix it?